src/components/NoToken.jsx

```diff
@@ -0,0 +1,13 @@
+import { useNavigate } from 'react-router-dom';
+
+const NoToken = () => {
+	const navigate = useNavigate();
+	return (
+		<div>
+			You do not have a list token! Please create one to start adding items.
+			<button onClick={() => navigate('/')}>Home</button>
```

Comment:
As I mentioned in our sync, it might be better to use React Router's `Link` component instead of using a button element and the `onClick` event so you don't need to call the hook.